tests/plotting/test_examples.py

```diff
@@ -0,0 +1,33 @@
+"""Unit-tests for :class:`geovista.examples`."""
+
+import geovista as gv
+from geovista.pantry import usgs_earthquakes
+
+
+def test_earthquakes(verify_image_cache):
+    """Test earthquakes."""
```

Comment:
```suggestion
    """Test earthquakes example."""
```

Reply:
@tkoyama010 Let's get this first example working in CI, then we can make this pattern more generic so that as we add new examples to `geovista.examples` or rename existing examples the tests will auto generate based on the examples available.

I can do that in a separate pull-request and ask you to review :+1:

Reply:
@bjlittle Thanks! Please set up a new PR after this one, as you probably know better than I do about that. I will be happy to review it.